packages/mobile/src/services/ReactNativeLogger.ts

```diff
@@ -33,6 +34,7 @@ export default class ReactNativeLogger {
     const sanitizedError =
       error && shouldSanitizeError ? this.sanitizeError(error, valueToPurge) : error
     const errorMsg = this.getErrorMessage(sanitizedError)
+    Sentry.captureException(error, { extra: { tag, message, errorMsg, source: 'Logger.error' } })
```

Comment:
i was wondering if we want to prevent errors from being sent to Sentry in dev mode? we can filter by envs in Sentry so we don't have to look at them but if we are paying by volume of captured events then probably not worth sending these?

Reply:
I think they aren't sent in dev mode because Sentry isn't initialized, so it no-ops. See `src/sentry/Sentry.ts#16`

Reply:
oh of course 🙈